phylo/src/optimisers/dna_model_optimiser.rs

```diff
@@ -58,48 +60,47 @@ impl<'a> DNAModelOptimiser<'a> {
 
     pub fn optimise_parameters(
         &self,
-        start_values: &DNASubstParams,
         optimise_freqs: FrequencyOptimisation,
     ) -> Result<(u32, DNASubstParams, f64)> {
```

Comment:
nit: consider using structs instead of tuples, as it isn't clear what is it returning

Reply:
Agreed, will be done in next PR where some of the optimisation is standardised.